Source/Shaders/PostProcessFilters/BloomComposite.glsl

```diff
@@ -0,0 +1,12 @@
+uniform sampler2D colorTexture;
```

Comment:
As I mentioned above, I think it is confusing that composite stages have composite stages.  Should we rename all the "composite" shader to be more precise, e.g., modulate, add, blend, etc.?